examples/E2EBert/news_classifier.py

```diff
@@ -57,22 +58,24 @@ def __getitem__(self, item):
         """
         review = str(self.reviews[item])
         target = self.targets[item]
-
-        encoding = self.tokenizer.encode_plus(
+        encoded = self.tokenizer.encode_plus(
             review,
             add_special_tokens=True,
-            max_length=self.max_length,
+            max_length=100,
             return_token_type_ids=False,
             padding="max_length",
             return_attention_mask=True,
             return_tensors="pt",
             truncation=True,
         )
+        # input_ids = torch.tensor([self.tokenizer.encode(
```

Comment:
Can you please remove the commented lines

Reply:
@shrinath-suresh  Sure.